Client/Frontend/Strings.swift

```diff
@@ -4488,6 +4488,132 @@ extension String {
         comment: "Tile title for Twitter")
 }
 
+// MARK: - Credit Card
+extension String {
+    public struct CreditCard {
+        // Settings / Empty State
+        public struct Settings {
+            public static let EmptyListTitle = MZLocalizedString(
+                "Settings.EmptyList.Title.v112",
+                tableName: nil,
```

Comment:
All new strings should have a `tableName`, this field is necessary for Messaging and although is not necessary at this point for Credit Card it might be in the future, see comment [here](https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Frontend/Strings.swift#L22)